js/huobi.js

```diff
@@ -736,8 +736,8 @@ module.exports = class huobi extends Exchange {
             request['size'] = limit; // 1-100 orders, default is 100
         }
         if (since !== undefined) {
-            request['start-time'] = since; // a date within 120 days from today
-            request['end-time'] = this.sum (since, 172800000); // 48 hours window
+            request['start-time'] = int (since); // a date within 120 days from today
```

Comment:
This is not the way you cast it. In fact we suggest to keep these type casting things outside of the lib in the userland, cause CCXT Manual documents that the since timestamp must be an integer in milliseconds.